requirements.txt

```diff
@@ -3,6 +3,8 @@ backcall==0.2.0
 cachetools==4.1.1; python_version ~= '3.5'
 certifi==2020.6.20
 chardet==3.0.4
+clize>=4.1.1
+python-dateutil>=2.8.1
```

Comment:
I suggest we pin all packages to exact version so there's no confusion caused by possible backwards-incompatible changes.